config/bootstrap.php

```diff
@@ -93,9 +93,9 @@
  * for a very very long time, as we don't want
  * to refresh the cache while users are doing requests.
  */
-if (!Configure::read('debug')) {
-    Configure::write('Cache._cake_model_.duration', '+1 years');
-    Configure::write('Cache._cake_core_.duration', '+1 years');
+if (Configure::read('debug')) {
+    Configure::write('Cache._cake_model_.duration', '+2 minutes');
+    Configure::write('Cache._cake_core_.duration', '+2 minutes');
```

Comment:
I'd not expect cache to be on at all when debug is on. If there is a good reason to have it on, 10 seconds sounds sane oob.